trl/models/utils.py

```diff
@@ -210,7 +216,27 @@ def clone_chat_template(
         pad_to_multiple_of=resize_to_multiple_of if resize_to_multiple_of is not None else None,
     )
 
-    return model, tokenizer
+    # After padding, the embedding matrix size may exceed the vocabulary size. Add dummy tokens to the tokenizer to
```

Comment:
turbo nit as I was initially confused why we were talking about padding:
```suggestion
    # After resizing, the embedding matrix size may exceed the vocabulary size. Add dummy tokens to the tokenizer to
```

Reply:
agree, did the same here: https://github.com/huggingface/trl/pull/3763/commits/098a33b53370f49e17deb320935282b4a17457d5